src/build_test.cc

```diff
@@ -473,7 +474,7 @@ struct FakeCommandRunner : public CommandRunner {
       max_active_edges_(1), fs_(fs) {}
 
   // CommandRunner impl
-  virtual bool CanRunMore() const;
+  virtual int CanRunMore() const;
```

Comment:
I think this should be returning a size_t.

You can't have a negative capacity, only 0 or more.

int also has platform specific limitations. it's 32bit, even on a 64bit machine.